src/renderer/domains/collectives/tracks/service.ts

```diff
@@ -235,6 +235,19 @@ export const getProposalTrack = (tracks: Track[], proposerMember: Member, wish:
   return 0;
 };
 
+function getRankFromTrack(track: Track): number {
```

Comment:
Maybe we could add a few comments here to make it clearer what these numbers represent